commit 894fede21953194c3abc3dfd8002df36be39d4c0
Author: Trent Mick <trentm@gmail.com>
Date:   2019-04-08T20:24:15-07:00 (6 months ago)
    
    OS-7720 vm and fw tests are creating VMs without cpu_caps which can break DAPI-using tests
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/src/fw/test/lib/vm.js b/src/fw/test/lib/vm.js
index e89c2f6f..5fad9c60 100644
--- a/src/fw/test/lib/vm.js
+++ b/src/fw/test/lib/vm.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016, Joyent, Inc. All rights reserved.
+ * Copyright 2019 Joyent, Inc.
  *
  * Test utilities for running vmadm commands
  */
@@ -52,7 +52,8 @@ function create(t, opts, callback) {
             autoboot: true,
             do_not_inventory: true,
             nowait: false,
-            ram: 128
+            ram: 128,
+            cpu_cap: 100
         };
 
         for (var p in opts.params) {
diff --git a/src/vm/tests/test-blocksizes.js b/src/vm/tests/test-blocksizes.js
index 7afa73fc..9f5a8191 100644
--- a/src/vm/tests/test-blocksizes.js
+++ b/src/vm/tests/test-blocksizes.js
@@ -1,4 +1,4 @@
-// Copyright 2017 Joyent, Inc.
+// Copyright 2019 Joyent, Inc.
 
 var async = require('/usr/node/node_modules/async');
 var cp = require('child_process');
@@ -23,6 +23,7 @@ test('create zone with root_recsize 64k', function(t) {
     var payload = {
         alias: 'test-blocksizes-' + process.pid,
         brand: 'joyent-minimal',
+        cpu_cap: 100,
         autoboot: false,
         image_uuid: image_uuid,
         do_not_inventory: true,
